app/src/main/java/com/conupods/OutdoorMaps/Services/OutdoorDirectionsService.java

```diff
@@ -107,7 +100,7 @@ private void setDirectionRoutes(DirectionsRoute[] routes) {
         if (this.mDirectionRoutes != null) {
             return mDirectionRoutes;
         } else {
-            return mDirectionRoutes;
+            return null;
```

Comment:
doesnt this mean its unnecessary

Reply:
yeah i i could indeed remove the whole if, ill wait to see if there other comments and ill push this later